react/rules/BEL.js

```diff
@@ -0,0 +1,59 @@
+import { isPastDate } from '../utils/dateRules'
```

Comment:
Two more things:

1 - Add an entry to the ChangeLog, [example](https://github.com/vtex/profile-form/commit/95374f4847d07dbc21bf041dd66e03db9f34c683#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed) 
2 - You also need this change to be made on the branch 3.x, so it works on IO

Reply:
Hi Felipe,

Entry in ChangeLog and [PR](https://github.com/vtex/profile-form/pull/106) for branch 3.x done